charts/argocd-image-updater/templates/deployment.yaml

```diff
@@ -9,6 +9,8 @@ spec:
   selector:
     matchLabels:
       {{- include "argocd-image-updater.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: {{ .Values.deploymentStrategy | default "Recreate" }}
```

Comment:
Let's use what we implemented in the notifications chart:
- https://github.com/argoproj/argo-helm/blob/master/charts/argocd-notifications/values.yaml#L27
- https://github.com/argoproj/argo-helm/blob/master/charts/argocd-notifications/templates/deployment.yaml#L9

Reply:
Good point ! I adjusted the strategy part and also updated the README with the helm-docs script.